src/test/java/com/demoqa/util/SingletonBrowser.java

```diff
@@ -0,0 +1,28 @@
+package com.demoqa.util;
+
+import org.openqa.selenium.WebDriver;
+
+public class SingletonBrowser {
```

Comment:
why this class?, reuse the existing class

Reply:
If I reuse the existing class I get this message, org.openqa.selenium.NoSuchSessionException